src/components/BaseStories/Link.stories.tsx

```diff
@@ -45,7 +43,7 @@ export const ExternalLink: Story = {
   render: (args) => <MockParagraph {...args} />,
 }
 
-export const LinkList: Story = {
+export const LinkList = {
```

Comment:
@TylerAPfledderer So if I understand correctly, if we're not supplying `args` (props) then we don't need to provide a type here?... but if we are, we should provide `StoryObj<typeof meta>` here, correct?

Reply:
@wackerow This probably needs clarity (including for myself!)

Because the `StoryObj` as a generic takes an optional prop type, it should be used on every story even if it's just 
```tsx
export const ExampleStory: StoryObj = {}
```
Because we still want to type the available options in the object.

I initially chose this removal because it didn't make sense to have `StoryObj<typeof meta>` when this is a complex story render that didn't need the args option.

So with the `LinkList` story, its type should be define as just `StoryObj` to maintain the base consistency.

---

Now with the choices in supplying a prop type to `StoryObj`, it can either be:
1. `StoryObj<typeof meta>`
2. `StoryObj<typeof ComponentName>`

With No. 1, [per the SB docs](https://storybook.js.org/docs/writing-stories/typescript#using-satisfies-for-better-type-safety) in conjunction with using `satisfies` to the `meta` object, you would apply this in the event you supply `args` to meta to be distributed to all the stories. This prevents TypeScript from throwing errors about missing required props at the story level if they were defined at the meta level.

No. 2 is for just the story level and not the meta level.

I have overlooked this specific detail, but I think `StoryObj<typeof meta>` also stays with consistency and the story file receives the most typing coverage.

> As an aside, not the details about custom args in the next section on that referenced page.

What do you think?